Containerfile

```diff
@@ -1,15 +1,39 @@
-FROM registry.access.redhat.com/ubi9/python-39:1-143.1697647134
+FROM registry.redhat.io/rhel9/python-311
```

Comment:
i think we're safer sticking w/ the ubi images, as they are not licensed/restricted compared to the rhel9 ones.

can we just use a ubi based image and then install/upgrade python within that image?  (yes i realize the build will take longer)

Reply:
I thought rhel9 and ubi9 were the same. The intent is default to UBI9 but pointing to the Python 3.11 image general name not a subversion of it

Reply:
my understanding is ubi is unencumbered, i don't think that's the case for rhel.